frontend/public/components/droppable-edit-yaml.tsx

```diff
@@ -31,6 +31,7 @@ const EditYAMLComponent = DropTarget(NativeTypes.FILE, boxTarget, (connectObj, m
 
 type DroppableEditYAMLProps = ResourceYAMLEditorProps & {
   allowMultiple?: boolean;
+  isOLSRedirected?: boolean;
```

Comment:
Maybe we can have a generic name to the prop not specific to OLS like `isCodeImportRedirect` or something? That way it can later be reused for something else apart from OLS as well.